wordpress-mobile/WordPress-Android
# Long press in Site Picker to remove self-hosted sites

Fixes #3313, covered by wordpress-mobile/WordPress-iOS#4379 for the iOS side.

The changes here are basically the same as the iOS counterpart, but I'm going to lay them out anyway.

1. Remove the `Remove Site` setting in self-hosted Site Settings. Functionality has been moved to Site Picker.

2. Long-pressing a self-hosted site in the Site Picker shows a dialog to remove the site.

![selfhostedlongpress](https://cldup.com/yzOQqa1WWG.png)

3. Long-pressing a WP.com site in the SitePicker is treated like Hide/Unhide settings option.

4. Minor text change to WP.com logout string; `Log out from WordPress.com` -> `Log out of WordPress.com`.

![logouttext](https://cldup.com/ORF9PuPsYp.png)

To test:
* Sign into WP.com
* Long press in the Site Picker to verify it begins Hide/Unhide action mode
* Add a self-hosted site
* Long press the self-hosted site and verify the dialog shows
* Select 'No' and make sure the site is not removed
* Long press it again and press 'Yes' to make sure it is removed
